README.md

```diff
@@ -39,3 +39,50 @@ timeout:
   begin_scenario: 120
   end_scenario: 120
 ```
+## Migration from built-in to plugin
```

Comment:
Whilst this isn't a hard and fast rule, I copied what they do on capybara in site_prism here. https://github.com/site-prism/site_prism/blob/main/UPGRADING.md

This to me feels a bit more tailored as you can have all of the help for getting around stuff (if you want to document it in detail).

Reply:
Yes, you right.
I knew when I did it that there were something not as smooth as it should be 😅 

I'll add a proper UPGRADING document 👍